src/Common/src/Internal/Cryptography/CryptoThrowHelper.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal static class CryptoThrowHelper
```

Comment:
Interop/[OS] is subdivided into (Win32) dll-based directories so I don’t see a place for it there as this helper isn’t tied to any Win32 dll.

Maybe under Internal\Cryptography\Windows, though this will no longer match the namespace then.